### ISSUE TITLE
重头训练的过程中loss突然变大，这是什么原因呢？

### ISSUE THREAD

**Issue by weishuoycbb** [yulunzhang/RCAN#98]
![image](https://user-images.githubusercontent.com/43461329/83984346-486c3c80-a967-11ea-90b6-349996a81cf2.png)

**Comment by John-Yao** [comment]
Hi, have you solved the problem? Thanks for you reply!

**Comment by qcccccc** [comment]
请问你是如何在训练的时候对于set 5，set 14这些数据集进行验证的？需要在启动的时候使用哪些指令？

**Comment by weishuoycbb** [comment]
> Hi, have you solved the problem? Thanks for you reply!

no

**Comment by BarCodeReader** [comment]
> > Hi, have you solved the problem? Thanks for you reply!
> 
> no

Hi,

do you mind to share your system configuration? I use 3090 and the latest pytorch & cuda to train RCAN, only 38.15 obtained on Set5.

I also directly use set5 as test set during training(same as yours)...

Thus may I know your system configuration? since I feel the different version of libraries really hurts the performance.

Thanks a lot!

**Comment by zejinwang** [comment]
> > > Hi, have you solved the problem? Thanks for you reply!
> > 
> > 
> > no
> 
> Hi,
> 
> do you mind to share your system configuration? I use 3090 and the latest pytorch & cuda to train RCAN,
> only **38.15** and **33.97** obtained on Set5 and Set14.
> 
> I also directly use set5 as test set during training(same as yours)...
> 
> Thus may I know your system configuration? since I feel the different version of libraries really hurts the performance.
> 
> Thanks a lot!

Hi,
 I encountered the problem of model non-convergence when training the RCAN model with an amplification factor of 2 from scratch, and the loss value has been oscillating around 1e29.  Could you tell me how to deal with this problem?

**Comment by raychl94** [comment]
你好，请问请问evaluateion 应该修改哪里的代码，把验证集改成 Set5 Set14 ......的数据集
